go/vt/health/health.go

```diff
@@ -74,41 +66,38 @@ func NewAggregator() *Aggregator {
 // the first error will be returned.
 // It may return an empty map if no health condition is detected. Note
 // it will not return nil, but an empty map.
```

Comment:
function-level comments (what are they called in Go? godocs?) need updating - this no longer returns a map.